utils/runcmd/constructClusterFromArg.go

```diff
@@ -0,0 +1,40 @@
+package runcmd
```

Comment:
These methods are strongly related to run and join cmd, and it is better to put them in cmd pkg.